new blog post: Announcing Plugins and Cloud Security with Falco (#545)
falcosecurity/falco-website: content/en/blog/announcing_plugins_and_cloud_security_with_falco.md

### Why Plugins?
Falco’s “runtime security” philosophy is based on some key concepts:
- Parse data in a streaming fashion to detect threats in real time
- Implement detection on top of an engine that is lightweight to run and easy to deploy 
- Offer a compact rule language that is quick to learn but flexible and expressive

This philosophy has proved to be very effective with system calls, and is the reason why Falco has thrived as a runtime security solution for containers.

Plugins extend the applicability of this philosophy to an endless number of new domains. One of these domains is cloud security.

### Runtime Security: a Better Option for Threat Detection in the Cloud
Cloud security is a fertile and constantly evolving space. When implementing cloud security you can choose among many different options. However, architecturally, most options fall in one of the following categories:

1. Tools that query the cloud APIs or watch cloud data stores to detect misconfigurations or vulnerabilities
2. Tools that stream cloud logs into a backend, index them and let you query them

> To detect threats in cloud-based software, Category 1 is not very useful. Polling is great to detect gaps and validate compliance, but lacks the real time nature required to detect threats and respond quickly. Category 2 is powerful, but it’s also tremendously expensive (especially in environments like the public cloud where tons of logs are produced) and not friendly to deploy and use.

Reviewer: ```suggestion
To detect threats in cloud-based software, Category 1 is not very useful. Polling is great to detect gaps and validate compliance, but lacks the real-time nature required to detect threats and respond quickly. Category 2 is powerful, but it’s also tremendously expensive (especially in environments like the public cloud where tons of logs are produced) and not friendly to deploy and use.
```

> I argue that the Falco runtime security approach is the ideal one. Falco consumes few resources and, most importantly, it analyzes the data in a streaming way. No need to perform expensive copies, no need to wait until the data is indexed. Falco looks at your data in real time and notifies you in seconds.

Reviewer: ```suggestion
I argue that the Falco runtime security approach is the ideal one. Falco consumes few resources and, most importantly, it analyzes the data in a streaming way. No need to perform expensive copies, no need to wait until the data is indexed. Falco looks at your data in real-time and notifies you in seconds.
```

Getting up and running with Falco takes only a few minutes, and adopting it for both cloud logs and system calls allows a unified approach to threat detection.

### What is the future for Falco?

> V0.31.0 comes with one plugin, Cloudtrail, but expect many more to come in the future. Our vision is making Falco the runtime policy engine for, well, everything. :-) We want to support all the clouds, and include more services from each of them. 

Reviewer: ```suggestion
V0.31.0 comes with one plugin, Cloudtrail, but expect many more to come in the future. Our vision is to make Falco the runtime policy engine for, well, everything. :-) We want to support all the clouds, and include more services from each of them. 
```

Stay tuned for announcements in the near future, and at the same time, please let us know if there is an area where you would like to see Falco in action in the future. Also, [writing your own plugin is easy](https://falco.org/docs/plugins/) and, as a community, we would love to consider your creative contribution. 

You can find us in the [Falco community](https://github.com/falcosecurity/community). Please feel free to reach out to us for any questions, suggestions, or even for a friendly chat!

If you would like to find out more about Falco:

* Get started in [Falco.org](http://falco.org/)
* Check out the [Falco project in GitHub](https://github.com/falcosecurity/falco).
* Get involved in the [Falco community](https://falco.org/community/).

> * Meet the maintainers on the [Falco Slack](https://kubernetes.slack.com/?redir=%2Farchives%2FCMWH3EH32).

Reviewer: ```suggestion
* Meet the maintainers on the [Falco Slack](https://kubernetes.slack.com/messages/falco).
```
